cores/esp8266/PolledTimeout.h

```diff
@@ -108,15 +133,25 @@ class timeoutTemplate
   
   bool expiredOneShot() const
   {
-    return checkExpired(millis());
+    return checkExpired(TimePolicyT::time());
   }
   
   timeType _timeout;
   timeType _start;
 };
 
-using oneShot = polledTimeout::timeoutTemplate<false>;
-using periodic = polledTimeout::timeoutTemplate<true>;
+using oneShotMs        = polledTimeout::timeoutTemplate<false>;
+using periodicMs       = polledTimeout::timeoutTemplate<true>;
+using oneShotMsYield   = polledTimeout::timeoutTemplate<false, YieldPolicy::YieldOrSkip>;
```

Comment:
Careful here, I named the policy YieldOrSkip on purpose due to the delay(0), meaning that the object will yield if it can (CONT) or do nothing otherwise (SYS). If you call the type periodicMs"Yield", it could conflict if ever there is a policy added that does not skip and forcibly yields (calls yield(), which will panic if it's ever called from SYS).